_harp/apply/_apply.md

```diff
@@ -1,75 +1,57 @@
-<h1 align='center'>APPLY</h1>
+<h1 align='center'>APPLY TO FOUNDERS & CODERS</h1>
 
-<h2 align='center'>APPLICATIONS FOR THE NEXT COHORT ARE NOW OPEN</h2>
+Founders & Coders welcomes applicants over 18 from all academic and professional backgrounds. We find that successful students are self-motivated, prepared to work long hours, and committed to collaborative working. Coding also requires a deep curiosity about how things work and a certain stubbornness in the face of repeated failure.
 
-<p align='center'>Submission deadlines for our 26th June - 13th October programme:</p>
+Before beginning a Founders & Coders programme, you will need to complete our prerequisites. We do not expect you finish this work before applying, but you will need to show us that you have made sincere effort to start this work and provide us with working usernames for GitHub, Free Code Camp, and Codewars in your application so that we can monitor your progress. Other recommended materials can be found [here](https://github.com/foundersandcoders/recommended-materials)..
 
-<p align='center'>**31st March 2017** (London)</p>
-<p align='center'>**30th April 2017** (Nazareth)</p>
+Applications are open for our next set of courses, which begin in late October. Please visit our LONDON and NAZARETH pages to learn more about the application and selection process in each location.
```

Comment:
Going for more visible buttons on this page, instead of embedding the links within a paragraph